### PR TITLE
Techdebt/remove auth prompt

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Overview

The original code was a fork from a private project where they planned on having many features for their original app (Breadwallet).  One of the ideas was extra layers of authorization.  

In Litewallet, one of the many complaints of our users is they can forget their device PIN and lose access to their Litecoin.  In this case, the user must remember that and the app PIN. 

This work is to remove that extra layer.  

## Bug Report
https://github.com/litecoin-foundation/litewallet-android/issues/241

## Definition of Done

- [ ] Remove all device authorizations
- [ ] Remove bio and fingerprint permissions  from the manifest
- [ ] Have a graceful experience for existing users
- [ ] Mitigate the AuthPrompt crashes: https://console.firebase.google.com/project/litewallet-beta/crashlytics/app/android:com.loafwallet/issues/e62bd3b2c450bdc67fa703c10e2ce5db?time=last-seven-days&types=crash&sessionEventKey=6729C3A80322000132F274DE0F4AE46D_2012139632503964489

### CODE REVIEW

**Comment by andhikayuana** [comment]
Hi @kcw-grunt do we have a plan to use biometric/fingerprint again?

**Comment by andhikayuana** [comment]
also I have an assumption that this crash maybe about not using Fragment from `androidx`, because app.fragment deprecated

File: app/build.gradle [hunk 1]
```diff
@@ -63,8 +63,8 @@ android {
         applicationId = 'com.loafwallet'
         minSdkVersion 31
         targetSdkVersion 34
-        versionCode 20241029
-        versionName "v2.12.0"
+        versionCode 20241101
+        versionName "v2.12.1"
```
Comment (andhikayuana) [inline app/build.gradle:67]: probably we can automate this things in the future

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -14,8 +14,6 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.READ_INTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
-    <uses-permission android:name="android.permission.USE_FINGERPRINT" />
-    <uses-permission android:name="android.permission.USE_BIOMETRIC" />
```
Comment (andhikayuana) [inline app/src/main/AndroidManifest.xml:18]: do you plan to use the fingerprint/biometric again in the future?

**Review by andhikayuana — APPROVED** [review]
LGTM

**Comment by kcw-grunt** [comment]
> also I have an assumption that this crash maybe about not using Fragment from `androidx`, because app.fragment deprecated

Yes, I think part it is we never managed the feature and it is 7 years old. So, that could be one problem.

**Comment by kcw-grunt** [comment]
> also I have an assumption that this crash maybe about not using Fragment from `androidx`, because app.fragment deprecated
More work for v2.13.X! : https://github.com/litecoin-foundation/litewallet-android/issues/261

**Review by josikie — APPROVED** [review]
Awesome. Thank you! @kcw-grunt

**Comment by kcw-grunt** [comment]
> Hi @kcw-grunt do we have a plan to use biometric/fingerprint again?
> 
> I have experience implementing passkeys in my previous company, did you know about passkeys? with passkeys, this is will not happen
> 
> > In Litewallet, one of the many complaints of our users is they can forget their device PIN and lose access to their
> > Litecoin. In this case, the user must remember that and the app PIN.
> 
> because it will use password/pin/pattern/face/fingerprint from the device / OS
No. But, when we are in the refactor / feature phases...we should discuss @andhikayuana

**Pull request merged**